docs/evaluate/temporal-cloud/aws-privatelink.mdx

```diff
@@ -114,7 +114,7 @@ When connecting, you must override the TLS server name to target your Namespace
       -servername <name>.<account>.tmprl.cloud \
       -cert /path/to/client.pem \
       -key /path/to/client.key \
-      vpce-XXXXXXXXX.vpce.amazonaws.com:7233 \
+      <DNS associated with VPC endpoint>:7233 \
```

Comment:
@fairlydurable @jsundai do we have a templated way of doing this? Maybe bold? Or all caps?

Reply:
bold won't work in a code block?
(note also I replaced it here because we were already using that value in the other code blocks)

Reply:
This is what I did for PrivateLink, i.e. what you're doing here. Let me try something experimental

  ```
  temporal workflow count \
      --address <DNS associated with VPC endpoint>:7233 \
      --tls-cert-path /path/to/client.pem \
      --tls-key-path /path/to/client.key \
      --tls-server-name <namespace>.<account>.tmprl.cloud \
      --namespace <namespace>
  ```

This is ugly, hard to maintain, and brittle (also I didn't bother going for exact corner radiuses and colors)

<img width="905" alt="image" src="https://github.com/user-attachments/assets/67134a5a-d052-4b81-8059-9666a4596dfc">

Reply:
But as a general rule, no, there is no bolding or highlights in code blocks except when externally sourced through snipsync

Reply:
So do we want the following ?
```suggestion
      <DNS ASSOCIATED WITH VPC ENDPOINT>:7233 \
```